packages/components/table/src/table-header/index.ts

```diff
@@ -141,118 +137,104 @@ export default defineComponent({
     } = this
     let rowSpan = 1
     return h(
-      'table',
+      'thead',
       {
-        border: '0',
-        cellpadding: '0',
-        cellspacing: '0',
-        class: ns.e('header'),
+        class: { [ns.is('group')]: isGroup },
       },
-      [
-        hColgroup(columns),
+      columnRows.map((subColumns, rowIndex) =>
         h(
-          'thead',
+          'tr',
           {
-            class: { [ns.is('group')]: isGroup },
+            class: getHeaderRowClass(rowIndex),
+            key: rowIndex,
+            style: getHeaderRowStyle(rowIndex),
           },
-          columnRows.map((subColumns, rowIndex) =>
-            h(
-              'tr',
+          subColumns.map((column, cellIndex) => {
+            if (column.rowSpan > rowSpan) {
+              rowSpan = column.rowSpan
+            }
+            return h(
+              'th',
               {
-                class: getHeaderRowClass(rowIndex),
-                key: rowIndex,
-                style: getHeaderRowStyle(rowIndex),
+                class: getHeaderCellClass(
+                  rowIndex,
+                  cellIndex,
+                  subColumns,
+                  column
+                ),
+                colspan: column.colSpan,
+                key: `${column.id}-thead`,
+                rowSpan: column.rowSpan,
```

Comment:
**Question:** I think this should be `rowspan` insteand of `rowSpan` since `colspan` is all lowercased. WDYT? Can you confirm that which way is the correct one?